packages/dashboard/src/components/elements/frame/frame.tsx

```diff
@@ -12,7 +12,6 @@ const Frame: React.FC<TFrameProps> = ({ src, alt, className }) => {
             src={src}
             alt={alt}
             width='200'
-            height='200'
```

Comment:
This file is useless since we don't have `Hall of fame` anymore 😄 , it's best to revert this change because it causes merge conflicts since i have removed this in my branch.

Reply:
sure thanks